File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ans;
        map<int,int> map1;
        for(int i=0;i<nums.size();i++){
            if (map1.find(target - nums[i]) != map1.end()) {
                return {map1[target - nums[i]] - 1, i};
            }
            else {
                map1[nums[i]] = i + 1;
            }

                    }
        return {};
    }
};

File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        mymap = dict()
        size = len(nums)
        for i in range(0,size):
            if target-nums[i] in mymap:
                return [mymap[target-nums[i]],i]
            else:
                mymap[nums[i]] = i            
            
        